Formula/mumps.rb

```diff
@@ -0,0 +1,201 @@
+class Mumps < Formula
+  desc "Parallel Sparse Direct Solver"
+  homepage "http://mumps-solver.org"
+  url "http://mumps.enseeiht.fr/MUMPS_5.1.2.tar.gz"
+  sha256 "eb345cda145da9aea01b851d17e54e7eef08e16bfa148100ac1f7f046cd42ae9"
+
+  depends_on :mpi => [:cc, :cxx, :f90, :recommended]
```

Comment:
please remove the `:recommended`

Reply:
@ilovezfs Could you explain why `:recommended` needs to be removed? My best guess is Homebrew would like to remove optional dependencies as many as possible so that most of formula can be installed using the bottles. 

However, I would like to migrate `ipopt` from `homebrew-science`, and it requires `mumps` built without `mpi`. In this case, what would be the best practice to formulate `mumps`?:
1. Remove `mpi` dependency from `mumps`. One concern would be some `mpi` dependent formula might want `mumps` built with `mpi`.
2. Leave `mpi` is an optional dependency of `mumps`.